### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by davemoore-**
This PR provides more context to the Getting Started page for Grafana.

This is an important page because it's the first actionable page in the Grafana docs and the first page many users will see when Googling "How do I get started with Grafana." Based on how the users may come to this page, I've updated this page to assume that the user might be starting completely from scratch: They haven't installed Grafana, they haven't decided whether they'll use Cloud or self-managed, and they don't know our terminology for plugins, data sources, or dashboards.

Prior to this PR, the page assumed that the user already has Grafana installed and is likely to start with Prometheus, InfluxDB, or MS SQL Server. I think this jumps too quickly into usage without a proper orientation to the prerequisites of getting started, which include: setting up Grafana, setting up a data source plugin, setting up a data source connection, and then setting up a dashboard.

### CODE REVIEW

File: docs/sources/getting-started/_index.md [hunk 1]
```diff
@@ -7,6 +7,29 @@ weight: 9
 
 # Get started
 
-This section provides guidance on how build your first dashboard after you have installed Grafana. It also provides step-by-step instructions on how to add a Prometheus, InfluxDB, or an MS SQL Server data source. Refer to [Data sources]({{< relref "../administration/data-source-management/" >}}) for a list of all supported data sources.
+Getting started with Grafana involves three steps:
+
+1. Setup Grafana
+2. Connect your first data source
```
Comment (jdbaldry) [inline docs/sources/getting-started/_index.md:13]: - It's preferable to use Markdown's auto-incrementing ordered lists. 
  To use the auto-incrementing lists, always use the prefix `1.`.

```suggestion
1. Connect your first data source
```

File: docs/sources/getting-started/_index.md [hunk 2]
```diff
@@ -7,6 +7,29 @@ weight: 9
...
+3. Create your first dashboard
+
+## Step 1. Setup Grafana
```
Comment (jdbaldry) [inline docs/sources/getting-started/_index.md:16]: - "Setup" is a noun, "Set up" is the intended verbiage.
```suggestion
## Step 1. Set up Grafana
```

File: docs/sources/getting-started/_index.md [hunk 3]
```diff
@@ -7,6 +7,29 @@ weight: 9
...
+
+You have two ways to setup Grafana:
+
+* [Setup Grafana on Grafana Cloud]("https://grafana.com/docs/grafana-cloud/quickstart/") without having to install it or run it yourself. This is the fastest way to get started and the easiest way to use Grafana.
```
Comment (jdbaldry) [inline docs/sources/getting-started/_index.md:20]: - "Setup" is a noun, "Set up" is the intended verbiage.
- It's preferable to put one sentence on each line for better future diffing with git.
- `prettier` prefers `-` for unordered lists over `*`.

```suggestion
- [Set up Grafana on Grafana Cloud]("https://grafana.com/docs/grafana-cloud/quickstart/") without having to install it or run it yourself.
  This is the fastest way to get started and the easiest way to use Grafana.
```

File: docs/sources/getting-started/_index.md [hunk 4]
```diff
@@ -7,6 +7,29 @@ weight: 9
...
+* [Setup Grafana on your own infrastructure]({{< relref "../setup-grafana/installation/" >}}) such as an operating system, Kubernetes cluster, or Docker container.
```
Comment (jdbaldry) [inline docs/sources/getting-started/_index.md:21]: - "Setup" is a noun, "Set up" is the intended verbiage.
- `prettier` prefers `-` for unordered lists over `*`.
```suggestion
- [Set up Grafana on your own infrastructure]({{< relref "../setup-grafana/installation/" >}}) such as an operating system, Kubernetes cluster, or Docker container.
```

File: docs/sources/getting-started/_index.md [hunk 5]
```diff
@@ -7,6 +7,29 @@ weight: 9
...
+
+## Step 2. Connect your first data source
+
+Grafana dashboards work by querying [data sources]({{< relref "../datasources/" >}}) that you connect to Grafana. You will need to [setup a data source]({{< relref "../administration/data-source-management/" >}}) connection before you can visualize data. Data sources are setup by [installing data source plugins]({{< relref "../administration/plugin-management/#install-grafana-plugins" >}})). You can [browse data source plugins](https://grafana.com/grafana/plugins/?type=datasource) from the plugin catalog.
```
Comment (jdbaldry) [inline docs/sources/getting-started/_index.md:25]: - It's preferable to put one sentence on each line for better future diffing with git.
```suggestion
Grafana dashboards work by querying [data sources]({{< relref "../datasources/" >}}) that you connect to Grafana.
You will need to [setup a data source]({{< relref "../administration/data-source-management/" >}}) connection before you can visualize data.
Data sources are setup by [installing data source plugins]({{< relref "../administration/plugin-management/#install-grafana-plugins" >}})).
You can [browse data source plugins](https://grafana.com/grafana/plugins/?type=datasource) from the plugin catalog.
```

File: docs/sources/getting-started/_index.md [hunk 6]
```diff
@@ -7,6 +7,29 @@ weight: 9
...
+
+1. Decide which data source you want to visualize. Browse the (data source plugin catalog](https://grafana.com/grafana/plugins/?type=datasource)) to see what you can connect with Grafana.
+2. Ensure the data source plugin is installed. See ([how to install data source plugins]({{< relref "../administration/data-source-management/" >}})).
+3. Connect the data source to Grafana. See ([how to setup a data source]({{< relref "../administration/data-source-management/" >}})), which configures an instance of a connection from Grafana to that data source.
+
+## Step 3. Create your first dashboard
+
+Once you have connected Grafana with one or more data sources, you can create a dashboard to visualize data from those data sources. This section provides guidance on how build your first dashboard after you have installed Grafana. It also provides step-by-step instructions on how to add a Prometheus, InfluxDB, or an MS SQL Server data source. Refer to [Data sources]({{< relref "../administration/data-source-management/" >}}) for a list of all supported data sources.
```
Comment (jdbaldry) [inline docs/sources/getting-started/_index.md:33]: - It's preferable to put one sentence on each line for better future diffing with git.

```suggestion
Once you have connected Grafana with one or more data sources, you can create a dashboard to visualize data from those data sources.
This section provides guidance on how build your first dashboard after you have installed Grafana.
It also provides step-by-step instructions on how to add a Prometheus, InfluxDB, or an MS SQL Server data source.
Refer to [Data sources]({{< relref "../administration/data-source-management/" >}}) for a list of all supported data sources.
```

File: docs/sources/getting-started/_index.md [hunk 7]
```diff
@@ -7,6 +7,38 @@ weight: 9
...
-This section provides guidance on how build your first dashboard after you have installed Grafana. It also provides step-by-step instructions on how to add a Prometheus, InfluxDB, or an MS SQL Server data source. Refer to [Data sources]({{< relref "../administration/data-source-management/" >}}) for a list of all supported data sources.
+Getting started with Grafana involves three steps:
+
+1. Set up Grafana
+1. Connect your first data source
+1. Create your first dashboard
+
+## Step 1. Set up Grafana
+
+You have two ways to set up Grafana:
```
Comment (chri2547) [inline docs/sources/getting-started/_index.md:18]:
```suggestion
Choose one of the following methods to set up Grafana:
```

File: docs/sources/getting-started/_index.md [hunk 8]
```diff
@@ -7,6 +7,38 @@ weight: 9
...
+
+- [Set up Grafana on Grafana Cloud]("https://grafana.com/docs/grafana-cloud/quickstart/") without having to install it or run it yourself.
+  This is the fastest way to get started and the easiest way to use Grafana.
+- [Set up Grafana on your own infrastructure]({{< relref "../setup-grafana/installation/" >}}) such as an operating system, Kubernetes cluster, or Docker container.
+
+## Step 2. Connect your first data source
```
Comment (chri2547) [inline docs/sources/getting-started/_index.md:24]:
```suggestion
## Connect your first data source
```

File: docs/sources/getting-started/_index.md [hunk 9]
```diff
@@ -7,6 +7,38 @@ weight: 9
...
+
+Grafana dashboards work by querying [data sources]({{< relref "../datasources/" >}}) that you connect to Grafana.
+You will need to [setup a data source]({{< relref "../administration/data-source-management/" >}}) connection before you can visualize data.
```
Comment (chri2547) [inline docs/sources/getting-started/_index.md:27]:
```suggestion
You must [set up a data source]({{< relref "../administration/data-source-management/" >}}) connection before you can visualize data.
```

File: docs/sources/getting-started/_index.md [hunk 10]
```diff
@@ -7,6 +7,38 @@ weight: 9
...
+Data sources are setup by [installing data source plugins]({{< relref "../administration/plugin-management/#install-grafana-plugins" >}})).
+You can [browse data source plugins](https://grafana.com/grafana/plugins/?type=datasource) from the plugin catalog.
```
Comment (chri2547) [inline docs/sources/getting-started/_index.md:29]:
```suggestion
From the plugin catalog, you can also [browse data source plugins](https://grafana.com/grafana/plugins/?type=datasource).
```

File: docs/sources/getting-started/_index.md [hunk 11]
```diff
@@ -7,6 +7,38 @@ weight: 9
...
+
+1. Decide which data source you want to visualize. Browse the (data source plugin catalog](https://grafana.com/grafana/plugins/?type=datasource)) to see what you can connect with Grafana.
+1. Ensure the data source plugin is installed.
+   See ([how to install data source plugins]({{< relref "../administration/data-source-management/" >}})).
+1. Connect the data source to Grafana.
+   See ([how to setup a data source]({{< relref "../administration/data-source-management/" >}})), which configures an instance of a connection from Grafana to that data source.
+
+## Step 3. Create your first dashboard
```
Comment (chri2547) [inline docs/sources/getting-started/_index.md:37]:
```suggestion
## Create your first dashboard
```

File: docs/sources/getting-started/_index.md [hunk 12]
```diff
@@ -7,6 +7,38 @@ weight: 9
...
+
+Once you have connected Grafana with one or more data sources, you can create a dashboard to visualize data from those data sources.
+This section provides guidance on how build your first dashboard after you have installed Grafana.
+It also provides step-by-step instructions on how to add a Prometheus, InfluxDB, or an MS SQL Server data source.
+Refer to [Data sources]({{< relref "../administration/data-source-management/" >}}) for a list of all supported data sources.
```
Comment (chri2547) [inline docs/sources/getting-started/_index.md:42]: @davemoore- , I would remove this link as it's covered above.

File: docs/sources/getting-started/_index.md [hunk 13]
```diff
@@ -7,6 +7,38 @@ weight: 9
...
+Grafana dashboards work by querying [data sources]({{< relref "../datasources/" >}}) that you connect to Grafana.
+You will need to [setup a data source]({{< relref "../administration/data-source-management/" >}}) connection before you can visualize data.
+Data sources are setup by [installing data source plugins]({{< relref "../administration/plugin-management/#install-grafana-plugins" >}})).
+You can [browse data source plugins](https://grafana.com/grafana/plugins/?type=datasource) from the plugin catalog.
+
+1. Decide which data source you want to visualize. Browse the (data source plugin catalog](https://grafana.com/grafana/plugins/?type=datasource)) to see what you can connect with Grafana.
+1. Ensure the data source plugin is installed.
+   See ([how to install data source plugins]({{< relref "../administration/data-source-management/" >}})).
+1. Connect the data source to Grafana.
+   See ([how to setup a data source]({{< relref "../administration/data-source-management/" >}})), which configures an instance of a connection from Grafana to that data source.
```
Comment (chri2547) [inline docs/sources/getting-started/_index.md:35]:
```suggestion
   See [how to set up a data source]({< relref "../administration/data-source-management/" >}}), which configures an instance of a connection from Grafana to that data source.
```

**Review by chri2547 — COMMENTED** [review]
Added some suggestions.

File: docs/sources/getting-started/_index.md [hunk 14]
```diff
@@ -7,6 +7,37 @@ weight: 9
...
+
+## Connect your first data source
+
+Grafana dashboards work by querying [data sources]({{< relref "../datasources/" >}}) that you connect to Grafana.
+You must [set up a data source]({{< relref "../administration/data-source-management/" >}}) connection before you can visualize data.
+You set up a data source by [installing a data source plugin]({{< relref "../administration/plugin-management/#install-grafana-plugins" >}})).
+From the plugin catalog, you can also [browse data source plugins](https://grafana.com/grafana/plugins/?type=datasource).
+
+1. Browse the [data source plugin catalog](https://grafana.com/grafana/plugins/?type=datasource) to decide which data source you want to visualize.
+1. Ensure the data source plugin is installed.
```
Comment (chri2547) [inline docs/sources/getting-started/_index.md:32]: @davemoore- This step and the following step land readers at the top of a very long data source management topic. Is there a specific section within that topic that we can link to?

**Review by chri2547 — COMMENTED** [review]
I have a question for you.

**Pull request closed without merging**